test/e2e/infra/secondary_network/prepare_cluster.sh

```diff
@@ -0,0 +1,358 @@
+#!/bin/bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+pushd $THIS_DIR
+
+_usage="Usage: $0 [--setup] [--cleanup]
+                  [--base-interface <interface name>] [--sriov-vf-count <count>] --host-name <name>
+
+Deploy Antrea CNI and its native secondary network prerequisites on an existing/running K8s cluster control node.
+
+        --setup                                 Deploy all the pre-requisite plugins and CRDs for secondary network configuration.
+        --cleanup                               Remove all the pre-requisite plugins and CRDs deployed for secondary network configuration.
+	--base-interface <interface name>       Network interface to create VFs.
+        --sriov-vf-count <count>                Number of virtual functions to be created on the base network interface.
+        --host-name <k8s control node name/IP>  K8s cluster control node's host name or IP address."
+
+_ssh_config_info="Prerequisite: Kubernetes(K8s) cluster is up and running at the local or remote server.
+
+Follow these steps to configure SSH information to access the kubernetes cluster control node.
+
+1) Update remote/local cluster control node information at your /home/<user>/.ssh/config
```

Comment:
could we actually support a custom ssh-config file (at a different path) using a command-line parameter?

Reply:
yes. we can do that.